### PR TITLE
Set up personal info pages

### PR DESCRIPTION
**Pull request by cehsu**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2820

#### Contact information
screen shot 2017-08-19 at 3 16 09 pm
screen shot 2017-08-19 at 3 19 32 pm

#### Secondary contact
screen shot 2017-08-19 at 3 20 34 pm
screen shot 2017-08-19 at 3 20 48 pm

#### Dependents
screen shot 2017-08-19 at 3 22 56 pm

#### Direct deposit 
screen shot 2017-08-19 at 3 23 18 pm

### CODE REVIEW

File: src/js/edu-benefits/1990-rjsf/config/form.js [hunk 1]
```diff
@@ -422,53 +433,95 @@ const formConfig = {
...
+              secondaryContact: {
+                type: 'object',
+                properties: {
+                  fullName: secondaryContact.properties.fullName,
+                  phone,
+                  'view:address': {
+                    type: 'object',
+                    properties: {}
+                  },
+                  sameAddress: secondaryContact.properties.sameAddress,
```
Comment (jbalboni) [inline src/js/edu-benefits/1990-rjsf/config/form.js]: I think you could put sameAddress and address inside 'view:address' and you would avoid having the empty object.

File: src/js/edu-benefits/1990-rjsf/config/form.js [hunk 2]
```diff
@@ -422,53 +433,95 @@ const formConfig = {
...
-        directDeposit: {
-          title: 'Direct deposit',
-          path: 'personal-information/direct-deposit',
+        directDeposit: _.merge(createDirectDepositPage(fullSchema1990), {
           uiSchema: {
-          },
-          schema: {
-            type: 'object',
-            properties: {
+            'ui:description': () => {
```
Comment (jbalboni) [inline src/js/edu-benefits/1990-rjsf/config/form.js]: We should put this description a helper file.

File: test/edu-benefits/1990-rjsf/config/directDeposit.unit.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-dom/test-utils';
+import { getFormDOM } from '../../../util/schemaform-utils';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1990-rjsf/config/form.js';
+
+describe('Edu 1990 dependents', () => {
```
Comment (jbalboni) [inline test/edu-benefits/1990-rjsf/config/directDeposit.unit.spec.jsx]: Need to change this description.

File: test/edu-benefits/1990-rjsf/config/dependents.unit.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-dom/test-utils';
+import { getFormDOM } from '../../../util/schemaform-utils';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1990-rjsf/config/form.js';
+
+describe('Edu 1990 dependents', () => {
```
Comment (jbalboni) [inline test/edu-benefits/1990-rjsf/config/dependents.unit.spec.jsx]: I also like testing the `depends` functions for pages with them directly. You can get it from the form config and call it whatever data makes sense.

**Review by jbalboni — APPROVED** [review]
Some minor things, but LGTM.

File: src/js/edu-benefits/1990-rjsf/config/form.js [hunk 5]
```diff
@@ -422,53 +433,95 @@ const formConfig = {
...
+              return (
+                <div>
+                  <p>VA makes payments only through direct deposit, also called electronic funds transfer (EFT). The only exception is for participants in the Post-Vietnam Era Veterans' Educational Assistance Program (VEAP).</p>
```
Comment (cvalarida) [inline src/js/edu-benefits/1990-rjsf/config/form.js]: Use a curly quote instead: `’` https://chrisbracco.com/curly-quotes/

**Review by cvalarida — APPROVED** [review]
Just one little thing.

File: src/js/edu-benefits/1990-rjsf/config/form.js [hunk 6]
```diff
@@ -31,10 +31,14 @@ import {
   benefitsRelinquishmentWarning,
   benefitsRelinquishmentLabels,
   benefitsRelinquishedDescription,
-  hasServiceBefore1977
+  directDepositDescription,
 } from '../helpers';
 
 import {
+  hasServiceBefore1977
+} from '../../utils/helpers.jsx';
```
Comment (cvalarida) [inline src/js/edu-benefits/1990-rjsf/config/form.js]: Did this not work without the extension?

File: test/edu-benefits/1990-rjsf/config/dependents.unit.spec.jsx [hunk 7]
```diff
@@ -1,22 +1,21 @@
 import React from 'react';
 import { expect } from 'chai';
 import ReactTestUtils from 'react-dom/test-utils';
-import { getFormDOM } from '../../../util/schemaform-utils';
 
-import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import { DefinitionTester, submitForm, getFormDOM } from '../../../util/schemaform-utils.jsx';
```
Comment (cvalarida) [inline test/edu-benefits/1990-rjsf/config/dependents.unit.spec.jsx]: Why is `submitForm` imported? Also, can we not use the extension?

**Pull request merged**